common/s2n-codec/src/decoder/checked_range.rs

```diff
@@ -76,3 +77,25 @@ impl fmt::Debug for CheckedRange {
         write!(f, "{}..{}", self.start, self.end)
     }
 }
+/// The following harnesses were automatically generated by an
+/// experimental tool developed by the Kani team.
+#[cfg(test)]
+mod bolero_harnesses {
+    use super::*;
+    #[test]
+    #[cfg_attr(kani, kani::proof)]
+    fn bolero_test_25_len() {
```

Comment:
do the tests really need `bolero_test` in them? It seems redundant given the module name is also `bolero_harnesses`.

Reply:
Do you propose just having `fn len_25() {}`? I'm fine with that

Reply:
I think that's probably _ok_, though I don't love the number. Optionally, you could use the callee type name to make it unique, e.g. `checked_range_len`.